vivisect/__init__.py

```diff
@@ -319,6 +319,12 @@ def getEmulator(self, **kwargs):
         emu = eclass(self, **kwargs)
         emu.setEndian(self.getEndian())
 
+        # PowerPC special case, if there are memory maps defined propagate those
+        # to the new emulator
+        maps = self.getMeta('PpcVlePages')
+        if maps is not None:
+            emu.setVleMaps(maps)
+
```

Comment:
we'll need to agnostify this somehow before it gets merged in.

Reply:
I really don't know the best way to handle this whole VLE memory page thing. it just seems to involve information from a bunch of different parts of vivisect that I don't know how to bring together in the right way.

Reply:
i'm thinking something like `archSetupEmu(self, vw, emu)`, kinda like your `archMarkupVW()`

Reply:
changed to use the PpcWorkspaceEmu initialization function to do any custom meta retrieval/setting.